pageserver/src/page_service.rs

```diff
@@ -697,7 +697,8 @@ impl Connection {
 
             // branch_create <branchname> <startpoint>
             // TODO lazy static
-            let re = Regex::new(r"^branch_create (\w+) ([\w@\\]+)[\r\n\s]*;?$").unwrap();
+            // TOOD: escaping, to allow branch names with spaces
+            let re = Regex::new(r"^branch_create (\S+) (\S+)[\r\n\s]*;?$").unwrap();
```

Comment:
@kelvich, just FYI, I changed your regexp slightly to allow "foo@0/12345678" syntax.